readme.md

```diff
@@ -39,6 +39,15 @@ Pass in an array of preferred ports:
 })();
 ```
 
+Use `makeRange` helper in case you need to select port from a range to generate an array of ports:
+
+```js
+(async () => {
+  console.log(await getPort({port: getPort.makeRange(3000, 3003)}));
```

Comment:
For examples sake, I would widen the range a bit. This example is not realistic. If you really want something close to 3000, you wouldn't stop it at 3003. I would make it `3000, 3010` at least.